location/templates/apartment.html

```diff
@@ -43,15 +45,39 @@
                 The apartment is in a walk-up building. It features an eat-in kitchen, hardwood floors and freshly
                 painted rooms. Good credit and proof of income are needed.</p>
 
-            <form class="d-flex justify-content-left mb-3">
-              <button class="btn btn-primary btn-md my-0 p" type="submit">Apply!</button>
-            </form>
+
             {% if show_claim_button %}
             <p>
               <a href="{% url 'claim' pk=apt.location.id suite_num=apt.suite_num %}" class="italic">Are you a tenant or a landlord of this apartment?</a>
             </p>
             {% endif %}
 
+          {% if apt.landlord %}
```

Comment:
nit: perhaps we should check to see if the the current user is not the landlord

Reply:
Good idea! Unless we want the landlord to send himself emails haha